### DISCUSSION TITLE
Still your intention to move the project forward?

### DISCUSSION THREAD

**Discussion by AtomicRobotMan0101** [mamolinux/system-monitoring-center#5] [General]
I see that a few people have made forks of the original project.

While I want involved in the coding, I did help with the doco, ideas, bugs and helped resolve a few issues.

It is a good project and I was saddened by the OG not to continue. I understand why this was the case.

BUT - now the project NEEDS to continue, for it was excellently envisaged and will be of great use now that the various Linuxen are exploding in popularity (due to recent Certain Decisions by MS)

So, may I ask, politely but firmly, is it your intention to continue the project and do you have the enthusiasm?

**Comment by hsbasu** [comment]
@AtomicRobotMan0101 I am really sorry for the delayed response. I never got a notification for this discussion. To answer your question, I definitely plan to maintain this app; at least in the foreseeable future. But there won't be any major upgrades. But I'll do bug fixes to the extent of my ability.
